libvips/foreign/jxlsave.c

```diff
@@ -114,65 +120,227 @@ typedef struct _VipsForeignSaveJxl {
 	void *runner;
 	JxlEncoder *encoder;
 
-	/* The current y position in the frame, page height,
-	 * total number of pages, and the current page index.
-	 */
-	int write_y;
-	int page_height;
-	int page_count;
-	int page_number;
-
-	/* VipsRegion is not always contiguous, but we need contiguous RGB(A)
-	 * for libjxl. We need to copy each frame to a local buffer.
+	/* Buffer scanlines to make a line of libjxl tiles.
 	 */
-	VipsPel *frame_bytes;
-	size_t frame_size;
+	VipsPel *scanline_buffer;
```

Comment:
This seems to be unused, as it's never set (but we try to `VIPS_FREEF()` it).

Reply:
Removed.